File: Fizz Buzz/FizzBuzz.cs
using System.Collections.Generic;

namespace LeetcodePracticeCsharpVersion
{
    class FizzBuzzProblem
    {
        public IList<string> FizzBuzz(int n)
        {
            var retList = new List<string>();
            for(int i = 1; i <= n; i++)
            {
                if(i % 3 == 0 && i % 5 == 0)
                {
                    retList.Add("FizzBuzz");
                }else if(i%3 == 0)
                {
                    retList.Add("Fizz");
                }else if(i % 5== 0)
                {
                    retList.Add("Buzz");
                }
                else
                {
                    retList.Add(i.ToString());
                }
            }
            return retList;
        }
    }
}

File: Fizz Buzz/FizzBuzz.py
class FizzBuzz(object):
    def fizzBuzz(self, n):
        retList = list()
        for i in range(1,n+1):
            if i % 3 == 0 and i % 5 ==0:
                retList.append("FizzBuzz")
            elif i % 3 == 0:
                retList.append("Fizz")
            elif i % 5 == 0:
                retList.append("Buzz")
            else:
                retList.append(str(i))
        return retList    